Customizing/global/plugins/Services/Repository/RepositoryObject/TalentAssessment/classes/Observations/SVGConverter.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace CaT\Plugins\TalentAssessment\Observations;
+require_once 'Services/Utilities/classes/class.ilUtil.php';
+
+class SVGConverter {
+	public function __construct($img_markup_string) {
```

Comment:
This looks strange. Why is the markup an argument to the constructor and not to the actual algorithm?